client/app/components/packages/equitable-development-reporting.hbs

```diff
@@ -12,7 +12,7 @@
       </p>
 
       <p>
-        The action(s) I am applying for are ULURP actions with an acquisition or disposition of land for:
+        I am applying for are ULURP actions with an acquisition or disposition of land for:
```

Comment:
Why does this copy look so off.... "I am appling for are " just sounds like something is missing?

Reply:
It is awkward. `are` probably should be dropped or leave `The action(s)` in. Aline wanted to drop that but perhaps didn't review the rest of the copy.

Reply:
the previous made more sense..

Reply:
gotcha. 

> It is awkward. `are` probably should be dropped or leave `The action(s)` in. Aline wanted to drop that but perhaps didn't review the rest of the copy.

Gotcha. Yeah i think you're right...

Reply:
They probably meant  " I am applying for ULURP actions with an acquisition or disposition of land for:" ?